Intent/app/src/main/java/com/example/dong/intent/Adapter.java

```diff
@@ -0,0 +1,53 @@
+package com.example.dong.intent;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+
+import com.bumptech.glide.Glide;
+
+import java.util.ArrayList;
+
+public class Adapter extends RecyclerView.Adapter<Adapter.ViewHolderImage> {
+
+    private ArrayList<String> listItem;
+    private Context mContext;
+
+    public Adapter(Context mContext, ArrayList<String> listItem) {
+        this.listItem = listItem;
+        this.mContext = mContext;
+    }
+
+    @NonNull
+    @Override
+    public ViewHolderImage onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
+        View view = LayoutInflater.from(mContext).inflate(R.layout.item_recyclerview, viewGroup, false);
+        return new ViewHolderImage(view);
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull ViewHolderImage viewHolderImage, int i) {
+        Glide.with(mContext).load(Constant.PATH + listItem.get(i))
+                .into(viewHolderImage.mImageView);
+    }
+
+    @Override
+    public int getItemCount() {
+        return listItem.size();
```

Comment:
check null, nên sử dụng toán tử 3 ngôi ở đây: listItem == null ? 0 : listItem.size